### DISCUSSION TITLE
OpenWRT support?

### DISCUSSION THREAD

**Discussion by samuk** [markqvist/Reticulum#179] [General]
I'm interested in creating a dual radio wifi mesh with 2x additional LoRa interfaces

**Software**

- [Libremesh dual radio](https://libremesh.org/howitworks.html)
- [OpenWRT ](https://openwrt.org/toh/hifive/unleashed)- Would this [Python under OpenWRT](https://openwrt.org/packages/pkgdata/python3) be sufficient to run reticulum? 

Once I have a Wifi mesh running I then just need to expose it to the Auto Interface?  https://reticulum.network/manual/interfaces.html

**Open hardware**

- RISC-V SOM https://github.com/antmicro/arvsom
- Carrier https://github.com/gumstix/PKG900000001400
- M.2 M > M.2 E https://www.amazon.co.uk/Ableconn-M2MN-150E-Converter-Board-Module/dp/B079NB8J3B
- Wifi https://www.wallystech.com/story-DR7915_Test_with_DR40x9_support_2.4G_5G_at_same_time.html
- LoRa https://www.olimex.com/Products/IoT/LoRa/LoRa868/open-source-hardware
- LoRa <> Pi https://www.olimex.com/Products/Modules/Adapters/RPi-UEXT/open-source-hardware

Possibly also look into [802.11ah](https://store.rokland.com/products/alfa-network-ahpi7292s-ieee-802-11ah-sub-1-ghz-module-in-raspberry-pi-hat-form-factor)

**Comment by 4c3e** [comment]
just want to mention OpenWRT support is mentioned under the "Universality" section of the [Reticulum Roadmap](https://github.com/markqvist/Reticulum/blob/master/Roadmap.md#universality)
